download-NC.py

```diff
@@ -80,36 +80,18 @@
 INTO TABLE {config['NC']['table']}
 CHARACTER SET latin1
 FIELDS TERMINATED BY ','
+ENCLOSED BY '"'
 LINES TERMINATED BY '\n'
-IGNORE 1 ROWS (@county_st, @voter_reg_num_st, @ignore, @last_name_st, @first_name_st, @middle_name_st, @race_st, 
-                @ethnicity_st, @gender_st, @age_st, @street_address_st, @city_st, @state_st, @zip_st, @dummy, @dummy, @dummy,
+IGNORE 1 LINES (county, voter_reg_num, @ignore, last_name, first_name, middle_name, race, 
```

Comment:
Whats the difference between `LINES` and `ROWS`?

Reply:
Not sure...I didn't do this part of it

Reply:
I changed it to see if it would fix the issues from earlier - seems interchangeable